src/components/DeleteQuestionModal.js

```diff
@@ -0,0 +1,43 @@
+import Modal from './Modal';
+
+class DeleteQustionModal extends Modal {
+  connectedCallback() {
+    this.attachShadow({ mode: 'open' });
+    this.render();
+    this.setComponentStyle(3);
```

Comment:
인자가 `zIndex` 값이라는게 `setComponentStyle` 이라는 메소드명만 보기에는 유추하기 어려운거 같아요

```suggestion
    this.setComponentStyle({ zIndex: 3 });
```
과 같이 객체로 인자를 내려주는건 어떨까요? (named parameter와 같은 개념으로)

Reply:
확실히 동동이 제시해준 코드가 더 좋아보입니다 ! 👍🔥